reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -534,7 +534,7 @@
 	 * <p><strong>Discard Support:</strong> This operator discards elements it internally queued for backpressure upon cancellation.
 	 *
 	 * @param sources The {@link Publisher} of {@link Publisher} to concatenate
-	 * @param prefetch the inner source request size
+	 * @param prefetch the number of Publishers to prefetch from the outer {@link Publisher}
```

Comment:
"Publishers" here is a little bit confusing since we request items, not Publishers, and then convert these items into Publishers with the provided mapper.

Reply:
ah yeah, agreed. maybe something along the lines of `number of elements to prefetch from the source, to be turned into inner Publishers` ?